qsirecon/interfaces/amico.py

```diff
@@ -226,8 +243,16 @@ def _list_outputs(self):
         base_metadata = {
             "Model": {
                 "Parameters": {
-                    "ParallelDiffusivity": self.inputs.dPar,
-                    "IsotropicDiffusivity": self.inputs.dIso,
+                    "dPar": {
+                        "value": self.inputs.dPar,
+                        "units": "mm^2/s",
+                        "bids_name": "Parallel Diffusivity constant",
```

Comment:
I'm not a fan of this. I'd prefer to (1) ensure that the units are whatever BIDS recommends, (2) have a single key-value pair, and (3) use the BIDS-compliant naming in the JSON. We can render the API documentation on RTD so users can see how dPar in the recon spec becomes dPar in the NODDI call and ParallelDiffusivity in the JSON.

Reply:
I suppose we can hold off on this until https://github.com/bids-standard/bids-specification/pull/2211/files#r2500747652 is resolved.